examples/src/main/scala/io/finch/eval/Main.scala

```diff
@@ -38,11 +38,12 @@ object Main extends App {
 
   val execute: Eval = new Eval()
 
-  val eval: Endpoint[Output] = post("eval" :: body.as[Input]) { i: Input =>
+  def eval: Endpoint[Output] = post("eval" :: body.as[Input]) { i: Input =>
     Ok(Output(execute[Any](i.expression).toString))
   } handle {
     case e: Exception => BadRequest(e)
   }
 
-  Await.ready(Http.server.serve(":8081", eval.toService))
+  def main(): Unit =
```

Comment:
Is there a reason we can't keep using `Main extends App` so we can avoid this `main` method?